Automattic/pocket-casts-ios
# Tracks: Add analytics to the Appearance settings

| 📘 Project: #154 | 🛫 Depends on: #307  |
|:---:|:---:|

- `settings_appearance_shown`: When the appearance settings is displayed to the user
- `settings_appearance_follow_system_theme_toggled`: When the user toggles the use system dark/light mode setting
- `settings_appearance_theme_changed`: When the user changes the theme and the use system theme option is disabled
- `settings_appearance_light_theme_changed`: When the user changes the theme to use while in light mode
- `settings_appearance_dark_theme_changed`: When the user changes the theme to use while in dark mode
- `settings_appearance_app_icon_changed`: When the user changes the app icon
- `settings_appearance_refresh_all_artwork_tapped`: When the user taps the refresh all artwork button
- `settings_appearance_use_embedded_artwork_toggled`: When the user toggles the user embedded artwork option

## To test

1. Go to the profile tab
2. Tap on the cog
3. Tap on Appearance
4. ✅ `🔵 Tracked: settings_appearance_shown`
9. Go through each setting and change its value
12. Check console and verify each setting has a corresponding event in console and the value displayed is the value that you selected
13. If the option changed is a switch then the event should be suffixed `_toggle` if it's a value change it should be suffixed with `_changed`


## Checklist
